ros2bag/ros2bag/verb/record.py

```diff
@@ -90,6 +90,13 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '--qos-profile-overrides-path', type=FileType('r'),
             help='Path to a yaml file defining overrides of the QoS profile for specific topics.'
         )
+        parser.add_argument(
+            '--resilient-storage-writing', action='store_true',
+            help='Indicate preference for avoiding data corruption in case of crashes, '
```

Comment:
Could clarify here that --resilient-storage-writing overrides only corresponding flags; additional flags from the config file remain in the pragmas list. I believe this is what is meant by "can override," but perhaps we can make that sentence a bit more explicit. For example, "When set, this flag will override corresponding settings in the config passed with --storage-config-file."

Reply:
Related, I'd like to see a test for the above behavior to ensure that overriding of params in a config file by using --resilient-storage-writing is occurring as expected.

Reply:
Applied - take a look at the unit test. I preferred to extend the sqlite storage API to enable querying of current settings (which seems useful anyway) because testing otherwise would be awkward.